nvflare/private/fed/app/simulator/simulator_runner.py

```diff
@@ -240,6 +250,15 @@ def setup(self):
             self.logger.info("Deploy the Apps.")
             self._deploy_apps(job_name, data_bytes, meta, log_config_file_path)
 
+            server_workspace = os.path.join(self.args.workspace, "server")
+            workspace = Workspace(root_dir=server_workspace, site_name="server")
+            custom_fobs_initialize(workspace)
+
+            decomposer_module = ConfigService.get_str_var(
+                name=ConfigVarName.DECOMPOSER_MODULE, conf=SystemConfigs.RESOURCES_CONF
+            )
+            register_ext_decomposers(decomposer_module)
```

Comment:
How would people use this auto register functionality?
They just write their own custom decomposers and put it inside app's custom folders?

Reply:
Yes, we will automatically scan the custom codes and register all the decomposers.